src/classes/PS_IntegrationServiceConfig_TEST.cls

```diff
@@ -161,17 +103,17 @@ public with sharing class PS_IntegrationServiceConfig_TEST {
         PS_IntegrationService.setConfiguration(testConfig);
 
         Test.startTest();
-        GE_PaymentServices service = new GE_PaymentServices();
+        PS_IntegrationServiceConfig.Service service = new PS_IntegrationServiceConfig.Service();
```

Comment:
The `PS_IntegrationServiceConfig.Service` replaced the configuration getter methods as `GE_PaymentServices` returned it before.